SPEC.md

```diff
@@ -653,7 +654,7 @@ You can show examples of """
 without worrying about escapes.
 ```
 
-or equivalently, `#"You can show examples of """\n    multi-line strings\n    """\nwithout worrying about escapes."#` as a Quoted String.
+or equivalently, `"You can show examples of """\n    multi-line strings\n    """\nwithout worrying about escapes."` as a Quoted String.
```

Comment:
The stuff between backticks is how you would represent the contents as a single line string. I think you’ll want to escape the quotes here.

Reply:
Fixed